widgets/creativecommons-widget.php

```diff
@@ -80,8 +80,18 @@ public function print_license_footer() {
 		public function widget( $args, $instance ) {
 			$title = apply_filters( 'widget_title', $instance['title'] );
 			echo $args['before_widget'];
-			echo $args['before_title'] . $title . $args['after_title'];
+
+			/*
+			* To prevent rendering an empty heading tag (and related margin)
+			* only render title when it exists (and isn't an empty string)
+			*/
+			$title_exists = $title && $title != "";
```

Comment:
I think you can use `!empty($title)` here, but please verify.

Also I am not sure if the `$title` is equal to `isset($title)`. Sorry, I am not that into php right now.

I could also consider this:
`$title_exists = isset($title) && !empty(trim($title));`

Reply:
Nice touch with `trim`. Thanks for pointing out how this code could be more idiomatic :-)

Reply:
I incorporated your suggestions into the pull request.